<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="abc-PC" name="RespondToQuery" tests="7" failures="1" timestamp="21 Nov 2017 10:48:05 GMT" time="116.728" errors="0">
  <testcase name="loginWithValidCredentials" time="82.347" classname="tests.RespondFunctionalityTest"/>
  <testcase name="sendQuery" time="12.446" classname="tests.RespondFunctionalityTest"/>
  <testcase name="verifyValidationMsgRespond" time="3.915" classname="tests.RespondFunctionalityTest"/>
  <testcase name="statusVerifyOnNotRespond" time="3.533" classname="tests.RespondFunctionalityTest"/>
  <testcase name="respondToQuery" time="3.509" classname="tests.RespondFunctionalityTest"/>
  <testcase name="verifyQueryLink" time="0.192" classname="tests.RespondFunctionalityTest"/>
  <testcase name="verifyStatusLink" time="0.27" classname="tests.RespondFunctionalityTest">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
expected [185154] but found [1]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [185154] but found [1]
	at steps.ActiveProjectsSteps.verifyingStatusLnkSteps(ActiveProjectsSteps.java:105)
	at tests.RespondFunctionalityTest.verifyStatusLink(RespondFunctionalityTest.java:79)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- verifyStatusLink -->
</testsuite> <!-- RespondToQuery -->
